gt/main.c

```diff
@@ -766,9 +968,18 @@ gt_proc(void *arg)
 	unsigned int block_idx = get_block_idx(gt_conf, lcore);
 	struct gt_instance *instance = &gt_conf->instances[block_idx];
 
+	uint64_t last_tsc = rte_rdtsc();
 	uint8_t port = get_net_conf()->front.id;
 	uint16_t rx_queue = instance->rx_queue;
 	uint16_t tx_queue = instance->tx_queue;
+	uint64_t frag_scan_timeout_cycles =
+		gt_conf->frag_scan_timeout_ms * rte_get_tsc_hz();
```

Comment:
The expression must be `round(gt_conf->frag_scan_timeout_ms / 1000. * rte_get_tsc_hz())` to correct the unit of `gt_conf->frag_scan_timeout_ms`; please notice the dot after 1000 since we want a float division.

Reply:
The expression `round(gt_conf->frag_scan_timeout_ms * rte_get_tsc_hz() / 1000.)` is slightly better when considering the precision.